### ISSUE TITLE
Mig-25PD Upgrade (multirole)

### ISSUE THREAD

**Issue by GreyDeer88** [PygmalionOfCyprus/cmo-db-requests#5475]
### DB Selector

DB3K

### Baseline

2377

### Hypothetical

Yes

### Name

Mig-25

### Length (m)

19.75

### Wingspan (m)

14

### Height (m)

6.3

### Empty Weight (kg)

20020

### Maximum Takeoff Weight (MTOW) (kg)

36720

### Payload Weight (kg)

Not known

### Crew

1

### Propulsion

TRDF R-15BD-300 Источник: https://testpilot.ru/rossiya/mikoyan/mig25pd/2/ Испытатели © www.testpilot.ru

### Rate of Climb (ft/min)

Same as Mig-25PD

### Service Ceiling (ft)

Same as Mig-25PD

### Military/Afterburner Speed (kt)

Same as Mig-25PD

### Cruise Speed (kt)

Same as Mig-25PD

### Supercruise

Yes

### Ferry Range (nm)

Same as Mig-25PD

### Endurance (min)

Same as Mig-25PD

### Takeoff / Landing Distance (m)

Same as Mig-25PD

### Fuel Capacity (kg)

Same as Mig-25PD

### Sensors

Same as Mig-25PD

### Mounts

Same as Mig-25PD

### Loadout Options

Same as Mig-25PD + newer russian missiles from the 1990s
+ Loadouts from Mig-25RB dbid 2434
+ loadouts from Mig-25BM dbid 2138
+ kh-31A, AD, P, PD

### Aircraft Codes

Night Navigation (Ferry, Air-to-Air, Air-to-Surface Missiles), Probe Refuelling

### Sources

None

**Comment by FrangibleCover** [comment]
Was this upgrade/conversion considered at any point? I suspect that the avionics for the radar, the very fancy bombing computer and the sensor suite to cue and aim the ARMs all took up the same space, cooling capacity and electrical capacity on the three different MiG-25 variants.

**Comment by GreyDeer88** [comment]
No, I have not been able to find anything but it seems unlikely that the
~1,000 airframes would remain static for the rest of their service life.
From the cutaways that I could find, it would be reasonable to expect that
as electronics got smaller then capabilities would be merged.

**Comment by PygmalionOfCyprus** [comment]
With no historical evidence that such a program was ever considered I'm going to close this ticket; we can think about "logical" upgrades for aircraft and make a billion hypothetical entries in the DB 'til the cows come home but I don't think that's necessarily a good move.
